cloudrds/cloudrds.go

```diff
@@ -0,0 +1,144 @@
+package cloudrds
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sync"
+
+	// we need mysql import because it's called in file, (but compile will complain because there is no display)
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/iotexproject/iotex-core/pkg/lifecycle"
+)
+
+var (
```

Comment:
This shouldn't be hard coded into the source code

Reply:
ok, let me put it in config